docs/config/README.md

```diff
@@ -1188,14 +1188,15 @@ The module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Variable             | Default         | Description                                                                 |
-| -------------------- | --------------- | --------------------------------------------------------------------------- |
-| `symbol`             | `"🐍 "`         | The symbol used before displaying the version of Python.                    |
-| `pyenv_version_name` | `false`         | Use pyenv to get Python version                                             |
-| `pyenv_prefix`       | `"pyenv "`      | Prefix before pyenv version display (default display is `pyenv MY_VERSION`) |
-| `scan_for_pyfiles`   | `true`          | If false, Python files in the current directory will not show this module.  |
-| `style`              | `"bold yellow"` | The style for the module.                                                   |
-| `disabled`           | `false`         | Disables the `python` module.                                               |
+| Variable             | Default         | Description                                                                   |
+| -------------------- | --------------- | ----------------------------------------------------------------------------- |
+| `symbol`             | `"🐍 "`         | The symbol used before displaying the version of Python.                      |
+| `pyenv_version_name` | `false`         | Use pyenv to get Python version                                               |
+| `pyenv_prefix`       | `"pyenv "`      | Prefix before pyenv version display (default display is `pyenv MY_VERSION`)   |
+| `python_binary`      | `python`        | Use this to change the python binary if, for example, you have an alias setup |
```

Comment:
I have just added this as an option here but should this go in an advanced configuration section?

Reply:
I would be in favor of having it in Advanced Configuration.

Reply:
Have added an Advanced Configuration section, (sorry for the force pushing but it's only the new commit that got forced)